app/src/main/java/com/github/se/orator/ui/settings/SettingsScreen.kt

```diff
@@ -137,6 +91,7 @@ fun SettingsScreen(
     userProfileViewModel: UserProfileViewModel,
     themeViewModel: AppThemeViewModel? = null
 ) {
+  val context = LocalContext.current
```

Comment:
I would recommend adding a context parameter to the screen and having it by default be = LocalContext.current.
There is a val context in MainActivity, and I found that passing it as a parameter for the screen makes the app performance better.

Reply:
Could you please provide some literature about the `LocalContext` performance impact ? I would gladly change it but I have never heard about this.